### PR TITLE
Filters by

### PR DESCRIPTION
**Pull request by SadiqZak**
Added Filter by ratings, category, type and sort by price

### CODE REVIEW

**Review by saketkum11 — APPROVED** [review]
- UI looks great.
- Easy to read code.
- Good choice of name convention.

File: src/Components/Card/Card.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+.card-footer{
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    border-top: 1px solid var(--background-grey-color);
+    /* border-bottom: 1px solid var(--background-grey-color); */
+    /* background-color: var(--white-color); */
```
Comment (Logan1x) [inline src/Components/Card/Card.css:34]: remove comments

File: src/Components/Card/Card.css [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+.card-footer-details{
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1rem 2rem 1rem 2rem;
+    font-size: 1.1rem;
+    font-weight: 500;
+    color: var(--button-color);
+    /* background-color: var(--white-color); */
```
Comment (Logan1x) [inline src/Components/Card/Card.css:57]: comments

File: src/Components/Chips/Chips.css [hunk 3]
```diff
@@ -0,0 +1,37 @@
+.chips-container{
+    display: flex;
+    flex-wrap: wrap;
+    gap: 1rem;
+    padding: 0.5rem;
+    background-color: var(--background-grey-color);
+    /* border-bottom:1px solid var(--icon-color) ; */
```
Comment (Logan1x) [inline src/Components/Chips/Chips.css:7]: here too

**Review by Logan1x — APPROVED** [review]
Overall good code. Just remove some unnecessary comments and you are good to go.

**Pull request merged**